tests/gui/test_02_import_data.py

```diff
@@ -112,7 +105,7 @@ def test_page_02_loads_maxquant_testfiles(mock_file_uploader: MagicMock):
     at.run()
 
     # User selects MaxQuant from the dropdown menu
-    at.selectbox(key="software").select("MaxQuant")
+    at.selectbox[0].select("MaxQuant")
```

Comment:
I know the goal was to remove unnecessary entries in the session state. However I see two reasons for using the key, i.e. keeping them in the session state: 1. we can address them in tests regardless of itnerface rearrangements, 2. we can set their value on loading settings. They will be part of the session_state with or without a key, but only with a key they can be addressed easily.

Reply:
agree.. done in #307
(added a trailing underscore to tell these widget keys apart from other session keys .. will get unified during the bookkeeping tasks)